packages/synapse-react-client/src/components/SynapseHomepageV2/SynapseInActionItem.tsx

```diff
@@ -0,0 +1,118 @@
+import {
+  Box,
+  Chip,
+  Link,
+  Typography,
+  useTheme,
+  useMediaQuery,
+  SxProps,
+} from '@mui/material'
+import React, { useEffect } from 'react'
+import ImageFromSynapseTable from '../ImageFromSynapseTable'
+import { EastTwoTone } from '@mui/icons-material'
+import { useInView } from 'react-intersection-observer'
+import { darkTextColor, homepageBodyText } from './SynapseHomepageV2'
+
+export type SynapseInActionItemProps = {
+  tableId: string
+  friendlyName: string
+  title: string
+  description: string
+  tags?: string[] | null
+  logoFileHandleId: string
+  link: string
+  primaryColor: string
+  secondaryColor: string
+  onInView: () => void
+}
+
+const mobileViewSxProps: SxProps = {
+  display: 'flex',
+  justifyContent: 'center',
+  flexDirection: 'column',
+  alignItems: 'center',
+  textAlign: 'center',
+}
+export const SynapseInActionItem: React.FunctionComponent<
+  SynapseInActionItemProps
+> = ({
+  tableId,
+  friendlyName,
+  title,
+  description,
+  tags,
+  logoFileHandleId,
+  link,
+  onInView,
+}) => {
+  const { ref, inView } = useInView({ threshold: 0.7 }) //do not report this is in view until the description is almost entirely shown (90%)
```

Comment:
I'm sure this has taken a lot of tweaking to get it to feel _just_ right
```suggestion
  const { ref, inView } = useInView({ threshold: 0.7 }) //do not report this is in view until the description is almost entirely shown (70%)
```